### ISSUE TITLE
Installing packmol results in a bunch of downgrades; necessary?

### ISSUE THREAD

**Issue by davidlmobley** [omnia-md/conda-recipes#697]
I just went to install `packmol` from Omnia in a relatively fresh py35 conda environment and got a notification that it required lots of downgrades (below). Is this necessary? It's not totally obvious to me why this would be; perhaps it's the requirement for `libgfortran ==1.0` in the meta.yaml? Is this needed? 

In my experience, packmol is basically a stand-alone CLI tool which doesn't interact with normal python modules, so I don't see any reason why using it should require downgrading so many other tools.

Here's the list of downgrades:
```
The following packages will be DOWNGRADED due to dependency conflicts:

    anaconda-navigator: 1.3.2-py35_0                                           --> 1.2.3-py35_0
    astropy:            1.2.1-np111py35_0                                      --> 1.1.2-np110py35_0
    bottleneck:         1.1.0-np111py35_0                                      --> 1.0.0-np110py35_1
    curl:               7.49.0-1                                               --> 7.45.0-0
    h5py:               2.6.0-np111py35_2                                      --> 2.6.0-np110py35_0
    hdf5:               1.8.17-1                                               --> 1.8.15.1-3
    libgfortran:        3.0.0-1                                                --> 1.0-0
    libnetcdf:          4.4.1-0                                                --> 4.3.3.1-3
    llvmlite:           0.15.0-py35_0                                          --> 0.11.0-py35_0
    matplotlib:         1.5.3-np111py35_1                                      --> 1.5.1-np110py35_0
    netcdf4:            1.2.4-np111py35_0                                      --> 1.2.2-np110py35_0
    numba:              0.30.0-np111py35_0                                     --> 0.26.0-np110py35_0
    numexpr:            2.6.1-np111py35_1                                      --> 2.5.2-np110py35_1
    numpy:              1.11.2-py35_0                                          --> 1.10.4-py35_2
    pandas:             0.19.1-np111py35_0                                     --> 0.18.1-np110py35_0
    pycurl:             7.43.0-py35_0                                          --> 7.19.5.3-py35_0
    pyqt:               5.6.0-py35_1                                           --> 4.11.4-py35_4
    pytables:           3.3.0-np111py35_0                                      --> 3.2.2-np110py35_1
    qt:                 5.6.2-2                                                --> 4.8.7-3
    qtconsole:          4.2.1-py35_1                                           --> 4.2.1-py35_0
    scikit-learn:       0.18.1-np111py35_0                                     --> 0.17.1-np110py35_2
    scipy:              0.18.1-np111py35_0                                     --> 0.16.0-np110py35_1
```

**Comment by davidlmobley** [comment]
As a follow-up to this, if I attempt to use the conda installed version on my cluster, I get an issue with too old a libgfortran version:
```
packmol: /usr/lib64/libgfortran.so.3: version `GFORTRAN_1.4' not found (required by packmol)
```


Conda list reveals that I indeed have libgfortran 1.0. So, if I "conda update libgfortran" I get this response:
```
Package plan for installation in environment /home/dmobley/anaconda3:

The following packages will be UPDATED:

    libgfortran: 1.0-0                                       --> 3.0.0-1
    openblas:    0.2.14-3                                    --> 0.2.14-4

The following packages will be DOWNGRADED due to dependency conflicts:

    packmol:     2016.06.09-1 http://conda.binstar.org/omnia --> 1.0.0-2  http://conda.binstar.org/omnia
```

So, this downgrades my packmol, but the resulting combination (of libgfortran 3 and an older packmol (?)) seems to work just fine.

**Comment by jchodera** [comment]
The downgrades are all from numpy 1.11 to 1.10. The [`packmol` recipe](https://github.com/omnia-md/conda-recipes/blob/master/packmol/meta.yaml) doesn't depend on numpy, so it must be one of the other dependencies. It doesn't look like it's [`libgfortran`](https://anaconda.org/anaconda/libgfortran/files) either since that also doesn't specify numpy.

**Comment by jchodera** [comment]
`libgfortran` versions 1 and 3 are very different, and provide dependencies for different fortran-compiled libraries. 

There may be a way for us to use a different fortran compiler to eliminate these fragile fortran library dependencies.

**Comment by jchodera** [comment]
[This thread](https://github.com/ContinuumIO/anaconda-issues/issues/686#issuecomment-194076923) seems to be related.

**Comment by jchodera** [comment]
[This post](https://github.com/ContinuumIO/anaconda-issues/issues/686#issuecomment-202322948) seems to suggest we want to specify `libgcc` instead of `libgfortran` in the recipe.

**Comment by davidlmobley** [comment]
@jchodera - did you need us to explore this, or were you going to?

**Comment by jchodera** [comment]
I won't be able to get to it before the weekend, so if you need it before then, please do try to update the recipe! You can always build and test locally by calling `conda build packmol; conda install --use-local packmol` locally.

**Comment by jchodera** [comment]
I won't be able to get to it before the weekend, so if you need it before then, please do try to update the recipe! You can always build and test locally by calling `conda build packmol; conda install --use-local packmol` locally.

**Comment by Lnaden** [comment]
Should be fixed by #864
